assign_rights/assemble.py

```diff
@@ -6,9 +8,12 @@
 class RightsAssembler(object):
     """docstring for RightsCalculator"""
 
-    def retrieve_rights(self):
+    def retrieve_rights(self, item, shell_list):
         """docstring for retrieve_rights"""
-    pass
+        identifier = item.get("identifier")
+        shell = RightsShell.objects.get(rights_id=identifier)
+        shell_list.append(shell)
+        return shell_list
```

Comment:
```suggestion
        return RightsShell.objects.get(rights_id=identifier)
```
If you follow my suggestion above, I think you can do this in one line. 

Also, @bonniegee can confirm, but you may want to simply target the model `pk`, because I think `rights_id` may be going away as a separate field.

Reply:
@helrond is correct, `rights_id` will be removed and we'll use `pk` to refer to rights shells